dwave/cloud/client.py

```diff
@@ -630,11 +631,15 @@ def get_solvers(self, refresh=False):
             self._all_solvers_ready = True
             return self._solvers
 
-    def solvers(self, name=None, qpu=None, software=None, vfyc=None,
-                flux_biases=None, anneal_schedule=None,
-                num_qubits=None, online=True, refresh=False):
+    def solvers(self, name=None, qpu=None, software=None, online=True,
+                refresh=False, **filters):
         """Returns a filtered list of solvers handled by this client.
 
+        Apart from a few inferred features (`name`, `qpu`, `software`, and
```

Comment:
"inferred" ?
Maybe drop this paragraph and move the "similar to Django QuerySet..." to the description of **filters. Having some explicit parameters in addition to a more general catch-all argument should not be confusing to users.